### PR TITLE
fix(orchestrator): add open api documentation to orchestrator readme

### PR DESCRIPTION
**Pull request by ydayagi**

### CODE REVIEW

**Review by pkliczewski — COMMENTED** [review]
/lgtm

File: plugins/orchestrator/README.md [hunk 1]
```diff
@@ -224,7 +224,8 @@ The plugin provides OpenAPI `v2` endpoints definition to facilitate communicatio
 
 In addition, by leveraging on OpenAPI spec, it is possible to generate clients and create CI steps.
 
-OpenAPI specification [file](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/openapi/openapi.yaml) is available in [orchestrator-common](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common).
+OpenAPI specification [file](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/openapi/openapi.yaml) is available in [orchestrator-common](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common).  
+OpenAPI specification documentation is available [here](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/generated/docs/index.adoc/index.adoc)
```
Comment (gciavarrini) [inline plugins/orchestrator/README.md:228]: It seems that the URL is "malformed" here `.../docs/index.adoc/index.adoc` (not your fault). Can you also fix the generation path?

Comment (ydayagi) [reply plugins/orchestrator/README.md:228]: the url seems fine. "https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/generated/docs/index.adoc/index.adoc"

Comment (gciavarrini) [reply plugins/orchestrator/README.md:228]: Probably I wasn't unclear. I know that the URL is correct. The documentation has been generated in that path, but it doesn't look nice. As I said is not your fault, I think I wrote the generation code. What I'm asking is: can you please fix the generation path as well?

Comment (gciavarrini) [reply plugins/orchestrator/README.md:228]: I'll fix this in future https://issues.redhat.com/browse/FLPATH-1625

**Review by gciavarrini — APPROVED** [review]

**Pull request merged**